frappe/email/receive.py

```diff
@@ -150,8 +151,15 @@ def connect_pop(self):
 				frappe.msgprint(_("Invalid User Name or Support Password. Please rectify and try again."))
 				raise
 
+	def folder_encode(self, folder):
+		# It is not yet known whether the folder name with quotes is passed to the method or not. 
+		# To select or get status a imap folder, you must send the name with quotes & encode for support in codings other than utf-8
+		if folder[0] == folder[-1] == '"':
+			folder = folder[1:-1]
+		return ImapUtf7.encode(f'"{folder}"')
```

Comment:
Can you cite some docs for this? libraries seem to be defaulting on `"INBOX"` without extra quotes.

Reply:
Thank you very much for your answer.
I've already lost all hope :(
1. A few lines below you can see the code that currently (before my changes) used quotes.
`res = self.imap.select(f'"{folder}"')`
2. https://datatracker.ietf.org/doc/html/rfc3501#section-5.1
> 1) Any character which is one of the atom-specials (see the Formal
> Syntax) will require that the mailbox name be represented as a
> quoted string or literal.
3. It seems like, theoretically, imaplib should insert quotes itself, but it doesn’t.
4. My changes have been tested and they work.